src/packages/core/components/input-image-cropper/image-cropper-focus-setter.element.ts

```diff
@@ -8,19 +9,28 @@ import {
 	nothing,
 	customElement,
 	property,
+	state,
 	query,
 } from '@umbraco-cms/backoffice/external/lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
 
 @customElement('umb-image-cropper-focus-setter')
 export class UmbImageCropperFocusSetterElement extends LitElement {
 	@query('#image') imageElement!: HTMLImageElement;
 	@query('#wrapper') wrapperElement!: HTMLImageElement;
 	@query('#focal-point') focalPointElement!: HTMLImageElement;
 
+	@state() private isDraggingGridHandle = false;
+
+	@state() private coords = { x: 0, y: 0 };
```

Comment:
Any better way to track `x` and `y` coords? The `onMove()` function returns `x` and `y` based on the rendered image size (not natural size, but client size).

It seems focal point `left` and `top` properties are readonly - so I couldn't set there directly.

But I think it is fine the track `x` and `y` coords and calculate this to updated focal point style in `#setFocalPoint()`.

Maybe the logic to update `focalPointElement` style should be wrapped in a private function, since it does that a few times.

```
this.focalPointElement.style.left = `calc(${left * 100}% - ${this.#DOT_RADIUS}px)`;
this.focalPointElement.style.top = `calc(${top * 100}% - ${this.#DOT_RADIUS}px)`;
```